Microsoft.HealthVault.Fhir.UnitTests/Samples/FhirExercise.json

```diff
@@ -0,0 +1,257 @@
+            "valueString": "Active"
+        },
+        {
+            "url": "http://healthvault.com/extensions/exercise-detail",
+            "extension": [
+                {
+                    "url": "exercise-detail-name",
+                    "valueString": "lap 1"
+                },
+                {
+                    "url": "exercise-detail-type",
+                    "valueCodeableConcept": {
+                        "coding": [
+                            {
+                                "system": "http://healthvault.com/vocabularies/",
+                                "version": "1",
+                                "code": "exercise-activities:swimming",
+                                "display": "swimming"
+                            }
+                        ]
+                    }
+                },
+                {
+                    "url": "exercise-detail-value",
+                    "valueQuantity": {
+                        "value": 30,
+                        "unit": "seconds",
+                        "system": "http://healthvault.com/vocabularies/duration-units/1",
+                        "code": "s"
+                    }
+                }
+            ]
+        },
+        {
+            "url": "http://healthvault.com/extensions/exercise-detail",
+            "extension": [
+                {
+                    "url": "exercise-detail-name",
+                    "valueString": "lap 2"
+                },
+                {
+                    "url": "exercise-detail-type",
+                    "valueCodeableConcept": {
+                        "coding": [
+                            {
+                                "system": "http://healthvault.com/vocabularies/",
+                                "version": "1",
+                                "code": "exercise-activities:swimming",
+                                "display": "swimming"
+                            }
+                        ]
+                    }
+                },
+                {
+                    "url": "exercise-detail-value",
+                    "valueQuantity": {
+                        "value": 28,
+                        "unit": "seconds",
+                        "system": "http://healthvault.com/vocabularies/duration-units/1",
+                        "code": "s"
+                    }
+                }
+            ]
+        },
+        {
+            "url": "http://healthvault.com/extensions/exercise-segment",
+            "extension": [
+                {
+                    "url": "exercise-segment-activity",
+                    "valueCodeableConcept": {
+                        "coding": [
+                            {
+                                "system": "http://healthvault.com/vocabularies/",
+                                "version": "1",
+                                "code": "exercise-activities:swimming",
+                                "display": "Swimming"
```

Comment:
wouldn't it make more sense to be [http://healthvault.com/vocabularies/exercise-activities](http://healthvault.com/vocabularies/exercise-activities) as url and then code "swimming"?